docs/hosting-a-server-with-northstar/guides/pg9182's-Docker-Installation-Guide.md

```diff
@@ -0,0 +1,84 @@
+---
+description: pg9182 has provided a complete docker image to run on Linux servers. This provides some server management tools and offers better performance. No physical GPU is needed. His github page can be found [here](https://github.com/pg9182/northstar-dedicated) This page is to provide basic instructions, more information and config options are found at pg9182's github. 
+---
+
+The aim of this guide is to show the steps to build a docker-compose container that starts on boot and to show basic configuration
+
+# <a name="Prerequisites">Prerequisites</a>
+
+| Requirement       | Description                                                                                        
+| ------------------| ---------------------------------------------------------| 
+| Kernal            | Linux 5.3+, but 4.9+ should work                         |
+| CPU               | x86_64, at least 3 cores minimum                         |
+| RAM               | 2GB (Typically peaks at 1.6GB)                           |
+| Disk              | 5GB                                                      |
+| Network           | Recommended at least 7-20Mbps up                         |
+| Docker            | Have Docker and Docker-compose installed on the machine  |
+
+
+# <a name="Installation">Installation</a>
+
+## Prep Titanfall Server files
+You will need to copy over your titanfall installation folder and place it on the Linux machine. Most of these files are for SP and therefore we can delete from the installation folder and prune the file size down to ~5GB. 
```

Comment:
```suggestion
You will need to copy over your Titanfall installation folder and place it on the Linux machine. Most of these files are for SP and therefore we can delete from the installation folder and prune the file size down to ~5GB. 
```

If we wanna be 100% correct, "Titanfall" should always be capitalised. Maybe even use "`Titanfall2` installation folder" ^^